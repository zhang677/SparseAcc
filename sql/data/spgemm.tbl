1|0|1|0.1
2|1|0|0.2
3|1|2|0.3
4|2|3|0.4
5|3|1|0.5