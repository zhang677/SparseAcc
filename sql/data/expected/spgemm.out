0|0|0.02
0|2|0.03
1|1|0.02
1|3|0.12
2|1|0.2
3|0|0.1
3|2|0.15